package.json

```diff
@@ -15,6 +15,7 @@
     "dist/",
     "MIGRATION.md"
   ],
+  "types": "dist/types/",
```

Comment:
Huh. I believe that when we discussed this during rewriting someone from the team said that we don't need types field here because we add the whole dist/ folder into the package.

Why did you decide to add the explicit types field? Something didn't work?

Reply:
We've discussed the PR privately.

`types` field is necessary here, because otherwise IDE doesn't use type declarations and tsc throws errors during bundling the app that uses Chronos.